source/robot_model/test/tests/test_kinematics.cpp

```diff
@@ -111,41 +111,41 @@ class RobotModelKinematicsTest : public testing::Test {
   state_representation::JointState dummy = state_representation::JointState(robot_name, 6);
-  EXPECT_THROW(franka->forward_geometry(dummy), exceptions::InvalidJointStateSizeException);
+  EXPECT_THROW(franka->forward_kinematics(dummy), exceptions::InvalidJointStateSizeException);
 }
 
 TEST_F(RobotModelKinematicsTest, TestForwardGeometryEE) {
-  EXPECT_EQ(franka->forward_geometry(joint_state).get_position(),
-            franka->forward_geometry(joint_state, "panda_link8").get_position());
+  EXPECT_EQ(franka->forward_kinematics(joint_state).get_position(),
+            franka->forward_kinematics(joint_state, "panda_link8").get_position());
 }
 
 TEST_F(RobotModelKinematicsTest, TestForwardGeometryInvalidFrameName) {
-  EXPECT_THROW(franka->forward_geometry(joint_state, "panda_link99"), exceptions::FrameNotFoundException);
+  EXPECT_THROW(franka->forward_kinematics(joint_state, "panda_link99"), exceptions::FrameNotFoundException);
 }
 
 TEST_F(RobotModelKinematicsTest, TestForwardGeometry) {
   for (std::size_t config = 0; config < test_configs.size(); ++config) {
-    state_representation::CartesianPose ee_pose = franka->forward_geometry(test_configs[config]);
+    state_representation::CartesianPose ee_pose = franka->forward_kinematics(test_configs[config]);
     EXPECT_TRUE(ee_pose.get_position().isApprox(test_fk_ee_expects.at(config).get_position()));
     EXPECT_TRUE(ee_pose.get_orientation().isApprox(test_fk_ee_expects.at(config).get_orientation()));
-    state_representation::CartesianPose link4_pose = franka->forward_geometry(test_configs[config], "panda_link4");
+    state_representation::CartesianPose link4_pose = franka->forward_kinematics(test_configs[config], "panda_link4");
     EXPECT_TRUE(link4_pose.get_position().isApprox(test_fk_link4_expects.at(config).get_position()));
     EXPECT_TRUE(link4_pose.get_orientation().isApprox(test_fk_link4_expects.at(config).get_orientation()));
   }
 }
 
 TEST_F(RobotModelKinematicsTest, TestForwardKinematics) {
   for (std::size_t config = 0; config < test_configs.size(); ++config) {
-    state_representation::CartesianTwist ee_twist = franka->forward_kinematic(test_configs[config]);
+    state_representation::CartesianTwist ee_twist = franka->forward_kinematics(test_configs[config]);
     EXPECT_TRUE(ee_twist.get_linear_velocity().isApprox(test_velocity_fk_expects.at(config).get_linear_velocity()));
     EXPECT_TRUE(ee_twist.get_angular_velocity().isApprox(test_velocity_fk_expects.at(config).get_angular_velocity()));
   }
 }
 
 TEST_F(RobotModelKinematicsTest, TestInverseKinematics) {
   for (std::size_t config = 0; config < test_configs.size(); ++config) {
-    state_representation::CartesianTwist ee_twist = franka->forward_kinematic(test_configs[config]);
-    state_representation::JointVelocities joint_twist = franka->inverse_kinematic(test_configs[config], ee_twist);
+    state_representation::CartesianTwist ee_twist = franka->forward_kinematics(test_configs[config]);
```

Comment:
The tests with `Kinematics` in their name should be changed to `Velocity` and use those methods I think

Reply:
Good catch. Test were actually working despite this because of the pose to twist implicit conversion which is quite interesting actually

Reply:
Agreed haha